packages/cli/src/commands/addons/create.ts

```diff
@@ -75,12 +75,32 @@ export default class Create extends Command {
     const argv = (restParse.argv as string[])
     // oclif duplicates specified args in argv
       .filter(arg => arg !== servicePlan)
+    const inferenceRegex = /heroku-inference.*/
```

Comment:
I would suggest at least removing the final `.*` (that's superfluous for this RegEx), and probably add an anchor at the beginning, meaning _any add-on service slug that starts with `heroku-inference`_.

```suggestion
    const inferenceRegex = /^heroku-inference/
```